compiler/src/iree/compiler/Codegen/Common/GPU/test/BUILD.bazel

```diff
@@ -6,8 +6,8 @@
 
 # Tests for common transforms.
 
-load("//build_tools/bazel:iree_lit_test.bzl", "iree_lit_test_suite")
 load("//build_tools/bazel:enforce_glob.bzl", "enforce_glob")
+load("//build_tools/bazel:iree_lit_test.bzl", "iree_lit_test_suite")
```

Comment:
One more question, what is the need for all of these changes to the bazel build files?

Reply:
I have had this local script for a while now that I always run before committing:

``` 
iree-presubmits() {
  python ${IREE_SOURCE_DIR}/build_tools/bazel_to_cmake/bazel_to_cmake.py
  (cd ${IREE_SOURCE_DIR} && ./build_tools/scripts/run_buildifier.sh)
}
```

It picked up those changes.